### DISCUSSION TITLE
NodoAmeを使ってソングボイスを喋らせた動画が、全くといっていいほど上がっていない

### DISCUSSION THREAD

**Discussion by cyatarow** [InuInu2022/NodoAme.Home#14] [General]
ソフトウェアの問題や不具合でもなく、こういう話題をIssueとして扱っていいのか疑問ですが・・・

NodoAmeの公開から3週間くらいが経ちました。
しかし、ソフト作者である@InuInu2022 さんの紹介動画([ニコニコ動画](https://www.nicovideo.jp/watch/sm39928066)、[YouTube](https://www.youtube.com/watch?v=lHshJSPfd-M))以外に、NodoAmeを使ってソングボイスを喋らせた動画が、全くといっていいほど投稿されていません。
[YouTubeではテスト的動画が1本存在する程度](https://www.youtube.com/results?search_query=nodoame&sp=CAI%253D)＜[動画リンク](https://www.youtube.com/watch?v=g_jj7EwFbY0)＞、[ニコニコ動画に至っては1本も投稿されていない](https://www.nicovideo.jp/search/NodoAme?sort=f&order=d)という状況です。

このソフト自体は、可不に限らず、つい最近発売された東北姉妹のソングボイスにも対応しているので、それを使って動画化するチャレンジャーが2～3人くらいいてもおかしくないと思うのですが、
新キャラや新技術には比較的素早く飛びつく(←私だけの印象かもしれませんが)トーク界隈が、ここまで無反応というのは不自然すぎる気がします。
一体なぜこんなことになっているのでしょう?

私はCeVIOユーザーではなく、いち視聴者ではありますが、可不以外のソングボイスの喋り方がどうなっているのか非常に気になっています。

**Comment by InuInu2022** [comment]
反応遅くなってすみません！さっき気が付きました！
ご提案ありがとうございます！

> 新キャラや新技術には比較的素早く飛びつく(←私だけの印象かもしれませんが)トーク界隈が、ここまで無反応というのは不自然すぎる気がします。
> 一体なぜこんなことになっているのでしょう?

このへんかな～とおもってます！
- 動画を作るのは大変！
- トークもソングも全部持ってる人はすくない
- 可不ちゃんアピールしかしていないので知られてない

### 動画を作るのは大変！

そもそも動画を作るのは大変なので、新しくNodoAmeをつかって動画をつくるのは大変だとおもいます。

たとえば、「かんしくん」というAviutlむけのツールがありますけど、
あれの設定ファイルは書くのが大変で、[公式のツール](https://oov.github.io/forcepser/)をつかうのが一番なのですが…
当然NodoAmeの事は考えられてないので簡単に設定ファイルをつくれない…ってことになっちゃってます。

ソフトウェアトークの動画は、どれだけ簡単でラクにたくさんセリフをつくれるか、
がキモなので（NodoAmeの隠しテーマです）、普段使ってるツールが使えないとなかなか動画にまでするのは大変だと思います！

とてもありがたいことなのですが、[この辺を乗り越えて動画を上げてくださっている方もいる](https://www.nicovideo.jp/watch/sm39965184)ので、色々ノウハウとかサポート系の機能を充実させればそのうち増えてくれるんじゃないかな～と思ってます！

### トークもソングも全部持ってる人はすくない

CeVIOユーザーでも両方やってる人はすくないので、そもそも人がすくない、って問題もあると思います！

東北三姉妹をしゃべらせたいならボイロをまず買うと思いますので、
ソングだけもってるけどしゃべらせたい、ってなるとかな～りレアになっちゃうかな～と思います。

### 可不ちゃんアピールしかしていないので知られてない

紹介動画は可不ちゃんメインなので、他のキャラの子たちのファンの人に伝わってない可能性があります！
なので紹介動画第2弾をつくらなきゃな…と思っています！

ちょうど[こういう企画もある](https://nico.ms/sm40038233)みたいなのでアピールできればな～、と思っています！

>可不以外のソングボイスの喋り方がどうなっているのか非常に気になっています。

[弦巻マキEnglish → IA Englishの動画はのせてる](https://youtu.be/XknZFUlhMWE)んですが、こんなかんじで
マニュアルにも載せた方がわかりやすいかもしれないですね！

ちなみにきりたんやイタコさんの場合は、歌ってる時の声と差が結構あるので、
二人とも若々しいような感じになります。
ささらちゃんは逆に（歌声が大人っぽいので）大人っぽくなります。
トークがあるキャラの場合は声質バーを調整したほうがイメージが合うのでその辺も調べて情報だせればな～、と思っています！

**Comment by InuInu2022** [comment]
issueからdiscussionへ移動しました。

ちなみに1年たって、定期的に投稿される方も出てきています！
